packages/env-loader/src/index.ts

```diff
@@ -8,53 +8,36 @@ export type EnvVars = {
   [key: string]: string
 }
 
-const getAllEnvVars = async (scriptName: string): Promise<EnvVars> => {
-  // CLI OPTIONS
-  const cliOptions = getCliOptions(scriptName)
+export const processEnv = async (command: string): Promise<void> => {
+  // CLI
+  const { args: cliArgs, options: cliOptions } = getCliOptionsAndArgs()
 
-  // ENV FILE VARS
+  // ENV FILE
   const envVars = getEnvFileVars(process.env.APP_ENV)
 
-  // For now, don't fetch CMS feature flags for tests.
-  // Will fail CI.
+  // CMS FEATURE FLAGS
+  let cmsFeatureFlags
   if (process.env.APP_ENV === 'test') {
-    return {
-      ...envVars,
-      ...cliOptions,
-    }
+    // For now, don't fetch CMS feature flags for tests. Will fail CI.
```

Comment:
not a blocker for this, just curious what failures you were seeing? is it related to fetching an env behind the VA network through GHA (gh-provided vs self-hosted runners), or something else?